src/celeritas/io/ImportElement.hh

```diff
@@ -8,20 +8,45 @@
 #pragma once
 
 #include <string>
+#include <vector>
 
 namespace celeritas
 {
 //---------------------------------------------------------------------------//
 /*!
  * Store element data.
+ *
+ * \c IsotopeIndex maps the isotope in the \c ImportData::isotopes vector.
  */
 struct ImportElement
 {
+    //!@{
+    //! \name type aliases
+    using IsotopeIndex = int;
```

Comment:
Should probably be `unsigned int` as in `ImportMatElemComponent` (maybe we should change that at some point?)